mcp/runtime/src/main/java/io/quarkiverse/langchain4j/mcp/runtime/config/ClaudeDesktopConfigSourceProvider.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkiverse.langchain4j.mcp.runtime.config;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiConsumer;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+import org.eclipse.microprofile.config.spi.ConfigSourceProvider;
+
+import io.quarkiverse.langchain4j.mcp.runtime.McpRecorder;
+import io.smallrye.config.PropertiesConfigSource;
+
+public class ClaudeDesktopConfigSourceProvider implements ConfigSourceProvider {
+    @Override
+    public Iterable<ConfigSource> getConfigSources(ClassLoader forClassLoader) {
+        if (McpRecorder.claudeConfigContents.isEmpty()) {
+            return List.of();
+        }
+
+        Map<String, String> configMap = new HashMap<>();
+        McpRecorder.claudeConfigContents.forEach(new BiConsumer<>() {
+            @Override
+            public void accept(String clientName, LocalLaunchParams localLaunchParams) {
+                if (!localLaunchParams.command().isEmpty()) {
+                    configMap.put(String.format("quarkus.langchain4j.mcp.%s.command", clientName),
```

Comment:
I would assume the client name might contain dots, so should we wrap `%s` in quotes?

Reply:
Yeah, that probably makes sense